maestro/pyproject.toml

```diff
@@ -19,11 +19,10 @@ jsonschema = "^4.23.0"
 docopt-ng = "^0.9.0"
 langchain-community = "^0.3.16"
 duckduckgo-search = "^7.3.0"
-crewai = "^0.100.1"
-crewai-tools = "^0.33.0"
```

Comment:
Given the version incompatibility, it might be worth considering if we can remove all framework-specific code and just implement generic agents. 

I can imagine this won't be the last time we run into version conflicts amongst all the various agentic frameworks...

Reply:
We might get there but I can see some specific agent framework code needed.